docs/learn-staking.md

```diff
@@ -70,78 +74,75 @@ The second point to note is that each validator candidate is free to name their
 
 ## Accounts
 
-There are three different accounts for managing your funds: `Stash`, `Controller` and `Session` accounts.
+There are two different accounts for managing your funds: `Stash` and `Controller`. The latter sets your [Session keys](learn-keys#session-keys).
 
-![staking](assets/NPoS/staking-keys.png)
+![staking](assets/NPoS/staking-keys_new.png)
```

Comment:
Would remove the Session altogether and just do something like this:
![image](https://user-images.githubusercontent.com/25483142/67389036-15249b00-f59a-11e9-810c-2b83d231c755.png)